doc/private_endpoint_connection_command_guideline.md

```diff
@@ -0,0 +1,84 @@
+    --name -n                       : The name of the private endpoint connection associated with the Key
+                                      Vault(Storage Account). Required if --id is not specified.
+    --resource-group -g             : Proceed only if Key Vault(Storage Account) belongs to the specified resource group.
+    --vault-name(--account-name)    : Name of the Key Vault(Storage Account). Required if --id is not specified.
+```
+- The `... private-endpoint-connection reject` command should look similar to the following, depending on which features are supported by the service.
+```
+Arguments
+    --rejection-description         : Comments for the rejection.
+    --id                            : The ID of the private endpoint connection associated with the Key
+                                      Vault(Storage Account). If specified --vault-name and --name/-n, this should be omitted.
+    --name -n                       : The name of the private endpoint connection associated with the Key
+                                      Vault(Storage Account). Required if --id is not specified.
+    --resource-group -g             : Proceed only if Key Vault(Storage Account) belongs to the specified resource group.
+    --vault-name(--account-name)    : Name of the Key Vault(Storage Account). Required if --id is not specified.
+```
+- The `... private-endpoint-connection show/delete` command should look similar to the following, depending on which features are supported by the service.
+```
+Arguments
+    --id                            : The ID of the private endpoint connection associated with the Key
+                                      Vault(Storage Account). If specified --vault-name and --name/-n, this should be omitted.
+    --name -n                       : The name of the private endpoint connection associated with the Key
+                                      Vault(Storage Account). Required if --id is not specified.
+    --resource-group -g             : Proceed only if Key Vault(Storage Account) belongs to the specified resource group.
+    --vault-name(--account-name)    : Name of the Key Vault(Storage Account). Required if --id is not specified.
+```
+
+#### Private Link Resource
+
+- The parent resource should expose a single command group called `private-link-resource` with one commands: `list`.
+- The `... private-link-resource list` command should look similar to the following, depending on which features are supported by the service.
+
+```
+Arguments
+    --vault-name(--account-name) [Required] : Name of the Key Vault(Storage Account).
+    --resource-group -g     : Proceed only if Key Vault belongs to the specified resource group.
+```
+- The output of `... private-link-resource list` should be a array instead of a dictionary.
+
+## Command Authoring
+
+Storage and keyvault modules both are good examples. Feel free to use them as reference.
```

Comment:
add links to source code in commits?

Reply:
sure. I'm waiting for storage PR and keyvault PR. I will add keyvault link first.